CANMotorUnit.c

```diff
@@ -269,3 +269,24 @@ void AssembleMaxPIDPWMPacket(CANPacket *packetToAssemble,
 uint16_t GetMaxPIDPWMFromPacket(CANPacket *packet){
     return DecodeBytesToIntMSBFirst(packet->data, 1, 2);
 }
+
+void AssembleLimSwEncoderBoundPacket(CANPacket* packetToAssemble,
+    uint8_t targetDeviceGroup,
+    uint8_t targetDeviceSerial,
+    uint8_t limSwNum,
+    int32_t encoderBound)
+{
+    packetToAssemble->id = ConstructCANID(PRIO_MOTOR_UNIT_SET_ENCODER_BOUND, targetDeviceGroup, targetDeviceSerial);
+    packetToAssemble->dlc = DLC_MOTOR_UNIT_ENCODER_BOUND;
+    int nextByte = WritePacketIDOnly(packetToAssemble->data, DLC_MOTOR_UNIT_ENCODER_BOUND);
+    PackShortIntoDataMSBFirst(packetToAssemble->data, limSwNum, nextByte);
+    nextByte++;
```

Comment:
Good catch, instead of using this pack method you should instead do:
```c++
packetToAssemble->data[nextByte] = limSwNum;
```

Since according to the packet spec it should only take up 1 byte, not 2.